src/scss/parse.js

```diff
@@ -3025,14 +3028,18 @@ function checkPseudoClass3(i) {
   if (l = checkSC(i)) i += l;
 
   if (i >= tokensLength) return 0;
-  if (tokens[i].value === '+' ||
-      tokens[i].value === '-') i++;
+
+  if (tokens[i].type === TokenType.PlusSign ||
+      tokens[i].type === TokenType.HyphenMinus) i++;
   else return 0;
 
   if (l = checkSC(i)) i += l;
 
+  if (l = checkInterpolation(i)) i += l;
+
   if (tokens[i].type === TokenType.DecimalNumber) i++;
-  else return 0;
```

Comment:
I'll have to revisit, but looking at that I have a feeling it may have slipped in unintentionally from when I was breaking down the issue.